src/main/java/com/stripe/model/TransferReversalCollection.java

```diff
@@ -54,9 +54,16 @@ public Reversal retrieve(String id, String apiKey) throws AuthenticationExceptio
     return retrieve(id, RequestOptions.builder().setApiKey(apiKey).build());
   }
 
+  /**
+   * Load a transfer reversal.
+   * @param id Id of the reversal
+   * @param options Further request options
+   * @return The transfer reversal
+   */
```

Comment:
So Checkstyle has this rule that any method with more than 2 lines of code has to have Javadoc. I guess comments count towards this... Thanks for adding the Javadoc :)